src/herder/test/HerderTests.cpp

```diff
@@ -1104,11 +1103,16 @@ testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps,
                                  uint64_t closeTime, SVUpgrades const& upgrades,
                                  bool sig) {
         txSet->sortForHash();
-        auto sv = StellarValue(txSet->getContentsHash(), closeTime, upgrades,
-                               STELLAR_VALUE_BASIC);
+        StellarValue sv;
         if (sig)
         {
-            herder.signStellarValue(root.getSecretKey(), sv);
+            sv = herder.makeStellarValue(txSet->getContentsHash(), closeTime,
+                                         upgrades, root.getSecretKey());
+        }
+        else
+        {
+            sv = StellarValue(txSet->getContentsHash(), closeTime, upgrades,
```

Comment:
actually we should remove this: the only place where we want to check that `BASIC` values are invalid is in the `SECTION("invalid")` where we should add a new mutation ("basic value")